agreement/voteTracker_test.go

```diff
@@ -521,7 +521,7 @@ func TestVoteTrackerFiltersDuplicateVoteOnce(t *testing.T) {
 			inputVotes[i] = helper.MakeValidVoteAcceptedVal(t, i, next, Val1)
 			expectedOutputs[i] = thresholdEvent{T: none}
 		case i == Num:
-			inputVotes[i] = voteFilterRequestEvent{RawVote: inputVotes[Num-1].(voteAcceptedEvent).Vote.R}
+			inputVotes[i] = voteFilterRequestEvent{RawVote: inputVotes[Num-1].(voteAcceptedEvent).Vote.R, Proto: protocol.ConsensusCurrentVersion}
```

Comment:
We could avoid passing proto since threshold isn't needed.

Reply:
Could you clarify that ? "isn't needed" as for testing, or isn't needed at all ?
I think that you'll want to have the threshold so that you could calculate the weight/threshold to determine if it's a high-stake vote. ( as described in the jira ticket.. )